libs/tenant_size_model/Cargo.toml

```diff
@@ -7,3 +7,5 @@ license = "Apache-2.0"
 
 [dependencies]
 workspace_hack = { version = "0.1", path = "../../workspace_hack" }
+log = { version = "0.4", features = ["std", "serde"] }
```

Comment:
I am surprised about this dependency, shouldn't we use `tracing`, while also being skeptical if the model needs any logging, didn't notice this before.

Reply:
I cannot see any log statements in the `tenant_size_model` being added, perhaps this is a debugging leftover? I usually use println's for quick-and-dirty logging, easier to spot as "should remove" later on.